### PR TITLE
docs(v2): improve cli docs

### PR DESCRIPTION
**Pull request by massoudmaboudi**
## Motivation

This part was missing in the documents and it is kinda useful to have this kinda referencing.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Run yarn run start:v2 and check that the page http://localhost:3000/ is rendered properly.

## Related PRs

No

### CODE REVIEW

**Comment by massoudmaboudi** [comment]
I added 'npm run swizzle' instead of 'docusaurus swizzle' because we use npx to install the documentation. So, when we use 'docusaurus swizzle', it shows an error that 'docusaurus: command not found'.

**Comment by massoudmaboudi** [comment]
I added the blog reference part while compiling, blog/2021-01-02-hello-there` will be converted to `blog/2021/01/02/hello-there` in the URL. For a normal user, this conversion is mysterious. But this explanation can remove this issue.

File: website/docs/using-themes.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Docusaurus Themes' components are designed to be replaceable. To make it easier
 To swizzle a component for a theme, run the following command in your doc site:
 
 ```shell
```
Comment (slorber) [inline website/docs/using-themes.md:111]:
```suggestion
```bash npm2yarn
```

will add npm/yarn tabs

File: website/docs/cli.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ Running the command will copy the relevant theme files to your site folder. You
 An example to use `--danger` flag let's consider the below code:
 
 ```bash
```
Comment (slorber) [inline website/docs/cli.md:119]:
```suggestion
```bash npm2yarn
```

Will add npm/yarrn tabs

File: website/docs/cli.md [hunk 3]
```diff
@@ -92,18 +92,18 @@ We highly discourage swizzling of components until we've reached a Beta stage. T
 
 :::
 
-Change any Docusaurus theme components to your liking with `docusaurus swizzle`.
+Change any Docusaurus theme components to your liking with `npm run swizzle`.
 
 ```shell
```
Comment (slorber) [inline website/docs/cli.md:97]:
```suggestion
```bash npm2yarn
```

**Review by slorber — APPROVED** [review]
thanks, will just add the npm2yarn tabs everywhere :)

**Review by slorber — CHANGES_REQUESTED** [review]
@massoudmaboudi please make smaller pull requests instead of one big that do multiple things.

I'm willing to merge:
- the fa translation file
- the CLI-related doc changes

But I'm not really willing to merge the "referencing blog posts" doc. 

If you made 3 separate PRs I would have been able to merge 2 of the 3 ;)

**Comment by slorber** [comment]
Extracted the fa translations to https://github.com/facebook/docusaurus/pull/4271

Will clean up the current PR from the "referencing" section and merge it, but feel free to resubmit it if you feel we should add more doc, but to me, this has nothing to do with the markdown features, it's just a regular link.

Here's a "backup" of your doc in case you find this useful:

```
## Referencing blog posts

If you want to reference a blog post file, you could use the name of the post you want to reference.

For example, if you are in `folder/doc1.md` and you want to reference `blog/2021-01-02-hello-there.md`:

```md
---
title: Hello There
---
```

You can add a slug to a blog post in its metadata. Consider another blog post named `blog/2021-01-08-bye-everyone.md`:

```md
---
title: Bye Everyone
slug: bye-everyone
---
```

I am referencing a blog post in `../../blog/2021/01/02/hello-there`.

Reference to another blog post with a slug in `../../blog/bye-everyone`.

:::caution Be careful that we should convert the address of the blog post without slug from:

`blog/2021-01-02-hello-there` to `blog/2021/01/02/hello-there` :::
```

**Comment by massoudmaboudi** [comment]
@slorber 
If you think it is not a good idea to add the blog referencing, then I will close the PR willingly.

**Comment by slorber** [comment]
I don't think it's a good idea, it's just a regular link nothing else. If we were to add this kind of doc, it couldn't be related to "markdown features" nor the blog, but be more generic, because it's just a link, and the link in md or in blog posts or in page is the same concept under the hood

**Pull request merged**